trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/get-selected-fee-record-ids-from-query.ts

```diff
@@ -0,0 +1,4 @@
+export const getSelectedFeeRecordIdsFromQuery = (selectedFeeRecordIdsQueryString: string | undefined) => {
+  const feeRecordIds = selectedFeeRecordIdsQueryString?.split(',').map((id) => parseInt(id, 10));
```

Comment:
What happens if the user types something random in as the query parameter e.g. 'abc'?

Reply:
Good point, I tested this initially in the browser and it all behaved okay but this would return `NaN` - I've updated this function to check for this and added unit tests to cover this case